packages/react-router-dom/__tests__/data-static-router-test.tsx

```diff
@@ -1,13 +1,9 @@
 import * as React from "react";
 import * as ReactDOMServer from "react-dom/server";
-import {
-  Route,
-  Outlet,
-  useLoaderData,
-  useLocation,
-  useMatches,
-} from "../index";
-import { DataStaticRouter } from "../server";
+import { Route, useLocation } from "react-router-dom";
+import { DataStaticRouter } from "react-router-dom/server";
+import { Outlet } from "react-router/lib/components";
+import { useLoaderData, useMatches } from "react-router/lib/hooks";
```

Comment:
Imports from sub-modules other than `react-router-dom/server` should probably be changed to import from the base package since these will no longer work in a real app anyway. All of these should probably be imported from `react-router-dom` (I'm assuming these imports came from a sloppy VSC auto-import).